docs(express/connect): Add performance options to express and connect (#9755)
getsentry/sentry-docs: docs/platforms/javascript/guides/connect/index.mdx

// const connect = require('connect');
// const Sentry = require('@sentry/node');

const app = connect();

> // Sentry must be initialized before doing anything else with it

Reviewer: ```suggestion
// Sentry must be initialized as early as possible
```

Sentry.init({
  dsn: "___PUBLIC_DSN___",

>   // Adding Performance Monitoring by setting tracesSampleRate

Reviewer: ```suggestion
  // Add Performance Monitoring by setting tracesSampleRate
```

  // Set tracesSampleRate to 1.0 to capture 100% of transactions
  // We recommend adjusting this value in production
  tracesSampleRate: 1.0,
});

> // The request handler must be the first middleware on the app

Reviewer: ```suggestion
// The Sentry request handler middleware must be added before any other handlers
```

app.use(Sentry.Handlers.requestHandler());

> // Performance Monitoring: creating a trace for every incoming request

Reviewer: ```suggestion
// Performance Monitoring: Create spans and traces for every incoming request
```

  res.end(res.sentry + "\n");
}
  // Adding Performance Monitoring by setting tracesSampleRate
  // Set tracesSampleRate to 1.0 to capture 100% of transactions
  // We recommend adjusting this value in production
  tracesSampleRate: 1.0,
});

// The request handler must be the first middleware on the app
app.use(Sentry.Handlers.requestHandler());
// Performance Monitoring: creating a trace for every incoming request
app.use(Sentry.Handlers.tracingHandler());

> connect(

Reviewer 2: is this example here just wrong, or old, or a different way to do the same thing, I wonder? 🤔

Author: It's wrong. `connect` does not accept a parameter and `connect.bodyParser()` and `connect.cookieParser()` is also not possible as those function are not available.

Reviewer 2: LOL, ok, great find xD

> // The error handler must be before any other error middleware

Reviewer: ```suggestion
// The Sentry error handler middleware must be registered before any other error middleware
```

>   // The error id is attached to `res.sentry` to be returned
>   // and optionally displayed to the user for support.

Reviewer: ```suggestion
  // The ID of error events captured by the Sentry error middleware is attached to `res.sentry`.
  // You can use this ID for debugging, or to correlate requests with errors in Sentry.
```
